tagcwebapp/static/index.htm

```diff
@@ -0,0 +1,33 @@
+<html>
+<head>
+  <title>TAGC - PL</title>
+  <link rel="stylesheet" type="text/css" href="css/style.css">
+  <link rel="stylesheet" type="text/css" href="css/jquery-ui.css">
+  <link rel="stylesheet" type="text/css" href="css/jquery-ui.structure.css">
+  <link rel="stylesheet" type="text/css" href="css/jquery-ui.theme.css">
+
+  <script src="js/jquery.js" type="application/javascript"></script>
+  <script src="js/jquery-ui.min.js" type="application/javascript"></script>
+  <script src="js/main.js" type="application/javascript"></script>
+</head>
+<body>
+
+<div id="header">
+
+</div>
+
+<div id="splashscreen">
+</div>
+
+<div id="container">
+  <div id="genome" class="mainGenome genome">
```

Comment:
Why is this area called genome. This naming seems confusing and it should be called minimap. That is more identifying.